aesara/tensor/basic.py

```diff
@@ -2246,22 +2247,17 @@ def addbroadcast(x, *axes):
 
 def unbroadcast(x, *axes):
     """
-    Make the input impossible to broadcast in the specified axes.
+    Make the input unknown to be broadcastable in the specified axes.
```

Comment:
It is only weird for me, because we are not really doing a negative assertion. Nothing will fail if the shape is actually 1 when it gets evaluated. It's just that Aesara will treat it as not being broadcastable when used in downstream `Op`s.

Reply:
We can add a comment to the docstring that says this `Op` only applies to the type-checking process.

In other words, we're saying "Specify (to the type-checking process) that the dimensions of `x` given by `axes` are non-broadcastable (i.e. not equal to one).".

Reply:
Otherwise, the phrasing "State that the input is not known to be broadcastable..." is also a little clearer.

Reply:
Yeah those sound better, thanks